package.json

```diff
@@ -55,7 +55,7 @@
     "test": "tools/scripts/test.sh",
     "test-with-temp-cloud": "tools/scripts/tests-with-temp-cloud.sh",
     "dtslint": "tools/scripts/ditslint.sh",
-    "lint": "eslint .",
+    "lint": "tools/scripts/lint.sh",
```

Comment:
I added lint.sh to prevent running eslint on node 6(which is not supported, and fails in Travis)